### PR TITLE
Update resource-validation.md

### PR DESCRIPTION
**Pull request by Krishnamohan-Yerrabilli**
### Why:

incorrect pronoun usage, incorrect article usage, and a few sentence issues...

### What's being changed:

Fixed "which is" to "that is" -> there is a pronoun problem here
Changed "is able to" to "can" -> "is able to" looks a bit wordy so changed  
Fixed "the rules that are available" to "the available rules" ->  shortened the phase
Fixed "gutter" to "the gutter" -> missing a determiner before it, so added for a better read
Changed "Github issue" to "GitHub issues" -> issue may not agree in number with the other words in the phrase

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] I completed the self-review checklist for content changes.

### CODE REVIEW

**Comment by Krishnamohan-Yerrabilli** [comment]
> Hello @Krishnamohan-Yerrabilli ,
> 
> Thank you for contributing to Monokle docs. 🙌🏻

looking more to contribute, that's what makes me happy :)

**Review by jfermi — COMMENTED** [review]
Looks good.
